src/platform-includes/sourcemaps/upload/esbuild/javascript.mdx

```diff
@@ -0,0 +1,58 @@
+Sentry uses [releases](/product/releases/) to match the correct source maps to your events. You can use the Sentry esbuild plugin to automatically create releases and upload source maps to Sentry when bundling your app.
+
+## Installation
```

Comment:
I recommend you make this change in all the files where you use this Installation/Configuration structure.
```suggestion
## Install
```